spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -126,22 +132,22 @@ function verifyChannelUpgradeErrorAbsence(
   counterpartyChannelIdentifier: Identifier,
 ) {
     client = queryClient(connection.clientIdentifier)
-    path = applyPrefix(connection.counterpartyPrefix, channelErrorPath(counterpartyPortIdentifier, counterpartyChannelIdentifier))
+    path = applyPrefix(connection.counterpartyPrefix, upgradeErrorPath(counterpartyPortIdentifier, counterpartyChannelIdentifier))
     client.verifyNonMembership(height, 0, 0, proof, path)
 }
 ```
 
-#### TimeoutPath
+#### Upgrade Timeout Path
 
-The timeout path is a public path set by the upgrade initiator to determine when the TRY step should timeout. It stores the `timeoutHeight` and `timeoutTimestamp` by which point the counterparty must have progressed to the TRY step. The TRY step will prove the timeout values set by the initiating chain and ensure the timeout has not passed. Or in the case of a timeout, in which case counterparty proves that the timeout has passed on its chain and restores the channel.
+The upgrade timeout path is a public path set by the upgrade initiator to determine when the TRY step should timeout. It stores the `timeoutHeight` and `timeoutTimestamp` by which point the counterparty must have progressed to the TRY step. The TRY step will prove the timeout values set by the initiating chain and ensure the timeout has not passed. Or in the case of a timeout, in which case counterparty proves that the timeout has passed on its chain and restores the channel.
 
 ```typescript
-function timeoutPath(portIdentifier: Identifier, channelIdentifier: Identifier) Path {
-    return "channelUpgrade/ports/{portIdentifier}/channelIdentifier/{channelIdentifier}/upgradeTimeout"
+function upgradeTimeoutPath(portIdentifier: Identifier, channelIdentifier: Identifier) Path {
```

Comment:
I added here `upgrade` as well for the same reason as for the error path.

Reply:
Same thing regarding the `channel` prefix.